h/app.py

```diff
@@ -42,6 +22,28 @@ def create_app(global_config, **settings):
 
     config.add_subscriber('h.subscribers.add_renderer_globals',
                           'pyramid.events.BeforeRender')
+    config.add_subscriber('h.subscribers.set_user_from_oauth',
+                          'pyramid.events.NewRequest')
+
+    config.add_tween('h.tweens.csrf_tween_factory')
+    config.add_tween('h.tweens.auth_token')
+
+    config.include(__name__)
+
+    app = config.make_wsgi_app()
+    app = permit_cors(app,
```

Comment:
I'd be happy to do that, but can we put it to a separate PR? It doesn't really defeat the point of CORS just because we use it liberally. We had the choice and I got to make it here, consciously. Our choice still doesn't include allowing credentials, which is the important vulnerability we'd like to avoid from cross origin requests. Allowing people to load our assets cross origin doesn't seem like a problem to me.